KEEP/bin/butch-core

```diff
@@ -224,9 +227,13 @@ start_build() {
 		echo "error: butch-merge failed"
 		exit 1
 	}
-        chmod +x $scrname
+	chmod +x $scrname
 	success=false
-        $scrname </dev/null > $logname 2>&1 && success=true
+	if allow_network "$1" ; then
```

Comment:
i think we need some kind of fallback here to check whether unshare works.
maybe if the command is install or rebuild, we do (once) `unshare --net true` and check return value, if it returns true we set UNSHARE_CMD to "unshare --net" else empty

Reply:
if it doesn't work, we could print a line like "warning: disabling network access protection" or something like that

Reply:
should a warning line be printed if the pkg requests network access?
I'm thinking no.

Reply:
no, but i think we should fix the few that do